### ISSUE TITLE
feat: Don't show "Disable auto updates" warning every time

### ISSUE THREAD

**Issue by ILoveOpenSourceApplications** [ReVanced/revanced-manager#1628]
### Bug description

When the user updates the app via revanced manager, even if they have done it before, they are still shown a warning screen saying to disable auto updates. I understand that this is significant for rooted users as the play store will try to update YouTube if it's not disabled. But is there any significance for this warning to be shown to non-rooted users? And if there is, I believe that it should only be shown once and once the user accepts the warning, it should not be shown to the user everytime they try to patch an app.

The warning message:
![Screenshot_20240109_151119_ReVanced Manager](https://github.com/ReVanced/revanced-manager/assets/117499019/6e91471d-0665-44c1-9c99-43efe33746ec)

### Version of ReVanced Manager and version & name of application you tried to patch

ReVanced Manager 1.18.0

### Installation type

Non-root

### Device logs

```shell
Nil
```

### Acknowledgements

- [X] This request is not a duplicate of an existing issue.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] The issue is solely related to the ReVanced Manager

**Comment by GlitchyPie** [comment]
I've never seen this warning, what are the conditions required for it to even show in the first place.

Also on 1.18.0

Non-root user.

**Comment by oSumAtrIX** [comment]
Yes, non-rooted users are significant. Your device may attempt to update the patched app by matching the package name of the patched app to a database. The warning should not be shown once; otherwise you'll forget it. At max, it can be a popup with a checkmark to not show again.

**Comment by ILoveOpenSourceApplications** [comment]
But isn't the package name changed by default for non-rooted users due to the GMS Core patch?

**Comment by Ushie** [comment]
ReVanced Manager isn't aware of whether or not the new package name exists as an entry in the PlayStore

**Comment by ILoveOpenSourceApplications** [comment]
> At max, it can be a popup with a checkmark to not show again.

I can agree with this as a solution. The user can agree to the statement and produce a pop-up saying never to show again after agreeing.

**Comment by Qaz-6** [comment]
I can't update the app because of this warning.In the latest version, when you click the install button, it says that the installation has been canceled.Apk export button not working
